scripts/cdn-cf.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+CDN_URL="flybywirecdn.com"
+CDN_PURGE_LINK="https://flybywirecdn.com/purgeCache?url=http://flybywirecdn.com"
+FILES=${1}
+CDN_DIR=${2:-"installer/config/test"}
```

Comment:
Is it expected to say "test" as the default? (I'm ok with it, but want to be sure)